src/Writers/HTMLWriter.jl

```diff
@@ -275,6 +276,20 @@ function render_search(ctx)
     end
 end
 
+
+# topbar for mobile
+# ------------------------------------------------------------------------------
+
+function render_topbar(ctx, navnode)
+    @tags div ul li span a i
+
+    page_title = string(mdflatten(pagetitle(ctx, navnode)))
+    topbarleft = ul[".float-left"](li(span(page_title)))
+    topbarright = ul[".float-right"](li(a[".btn", :href => "#"](i[".fa .fa-bars", Symbol("aria-hidden") => "true"])))
```

Comment:
Why is the `aria-hidden` here? Based on [this article](https://www.paciellogroup.com/blog/2012/05/html5-accessibility-chops-hidden-and-aria-hidden/) it seems that `aria-hidden` is not very useful and, if anything, we should use HTML5's `hidden` attribute.

Also, why use `ul`s here, this does not feel like a list to me? A simple `<div id="topbar">{PAGE_TITLE}<a class="btn fa fa-bars"></a></div>` ought to do the trick here.